Mathlib/Analysis/SpecialFunctions/Pow/Real.lean

```diff
@@ -513,6 +513,19 @@ lemma rpow_mul_intCast (hx : 0 ≤ x) (y : ℝ) (n : ℤ) : x ^ (y * n) = (x ^ y
 /-! Note: lemmas about `(∏ i in s, f i ^ r)` such as `Real.finset_prod_rpow` are proved
 in `Mathlib/Analysis/SpecialFunctions/Pow/NNReal.lean` instead. -/
 
+theorem prod_div_rpow_distrib {ι : Type*} {s : Finset ι} {f g h : ι → ℝ}
+    (hf : ∀ x ∈ s, 0 ≤ f x) (hg : ∀ x ∈ s, 0 ≤ g x) :
+    ∏ x in s, (f x / g x) ^ (h x) = (∏ x in s, (f x) ^ (h x)) / (∏ x in s, (g x) ^ (h x)) := by
+  revert hf hg
+  apply Finset.cons_induction_on (p := fun s => (∀ x ∈ s, 0 ≤ f x) → (∀ x ∈ s, 0 ≤ g x) →
+    ∏ x in s, (f x / g x) ^ (h x) = (∏ x in s, (f x) ^ (h x)) / (∏ x in s, (g x) ^ (h x)))
+  · simp only [implies_true, Finset.prod_empty, ne_eq, one_ne_zero, not_false_eq_true, div_self]
+  · simp only [Finset.prod_cons, Finset.mem_cons]
+    intro x s _ ih hf hg
+    rw [ih, div_rpow, div_mul_div_comm] <;>
+    intros <;> (first | apply hf | apply hg) <;>
+    first | (left; trivial) | (right; trivial)
```

Comment:
The proof simplifies to this, as it's basically just the combination of `Finset.prod_div_distrib` and `Real.div_rpow`.
```suggestion
  rw [← Finset.prod_div_distrib]
  congr! 1
  rename_i x hx
  rw [div_rpow (hf x hx) (hg x hx)] 
```

Reply:
I think you can use `congr! 1` with x hx` there